pkg/migrations/sql/idempotent/base.sql

```diff
@@ -1693,31 +1693,56 @@ $$;
 
 --------------------------------- Views --------------------------------
 
+
+
+
+
```

Comment:
Why?

Reply:
that was for debugging, for faster access to the function. updated it.